src/app/savings/saving-account-actions/savings-account-transactions/savings-account-transactions.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="container">
+  <h2 mat-title *ngIf='transactionType.withdrawal'>Withdraw Money From Saving Account</h2>
+  <h2 mat-title *ngIf='transactionType.deposit'>Deposit Money To Saving Account</h2>
+
+  <form [formGroup]="savingAccountTransactionForm" (ngSubmit)="submit()">
+
+    <mat-card-content fxLayout='column'>
+
+      <mat-form-field>
```

Comment:
@abhaychawla I went with single column layout because by default there are only 4 fields  and add button to add payment details so two-column layout was not appearing as clean as single column layout. If I make this a two-column layout were do you suggest to keep the add payment details button.

Reply:
@Jov03 Alright it's okay. Let keep it this way. Should we reduce the mat-card width like the other sections containing single-column layout?

Reply:
sure